Bulldozer.CSV/Maps/Group.cs

```diff
@@ -1268,7 +1270,7 @@ public void BulkInsertGroupTypeRoles( RockContext rockContext, List<GroupMemberI
                         var newGroupTypeRole = new GroupTypeRole
                         {
                             GroupTypeId = groupTypeCache.Id,
-                            Name = textInfo.ToTitleCase( roleName.Left( 100 ) ),
+                            Name = roleName,
```

Comment:
@guymaness2 because of this change can you please remove Line 1258? It should no longer be used anywhere?